panics.go

```diff
@@ -117,8 +117,17 @@ func CaptureHTTPRouterHandler(h httprouter.Handle) httprouter.Handle {
 }
 
 // Capture will publish any errors
-func Capture(err string, message string) {
-	publishError(errors.New(err), []byte(message), false)
+func Capture(err string, message ...string) {
+	var tmp string
+	for i, val := range message {
+		if i == 0 {
+			tmp += val
+		} else {
+			tmp += fmt.Sprintf("\n\n%s", val)
```

Comment:
\n is not gonna work, because we'll marshal before sending the request. so it'll be escaped.